### DISCUSSION TITLE
【求助】我的绳网总是会被电脑检测为危险软件然后自动删除，怎么办？

### DISCUSSION THREAD

**Discussion by Nie7bai** [share121/inter-knot#5148] [委托]
之前就因为这个问题小半个月没有登上绳网<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by yzhfat** [comment]
关闭自带防火墙。或者添加该程序为信任程序

**Comment by Bailinghai** [comment]
把绳网加入白名单就可以了

**Comment by TNCNET** [comment]
先把主程序/相关目录加入白名单。之后再试试。

**Comment by Yizakl** [comment]
# 关闭Windows安全中心

**Comment by Nie7bai** [comment]
好的，非常感谢，我这就去试试<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >
